test/checkbox.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { test, Page } from "bigtest";
+import { Body, CheckBox as Interactor } from "../src/index";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+import { render } from "./helpers";
+
+export default test("Checkbox")
+  .step(Page.visit("/"))
+  .child("filter", (test) =>
+    test
+      .child("by locator", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="locator" control={<Checkbox />} />))
+          .assertion(Interactor("locator").exists())
+      )
+      .child("by checked", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="checked" checked control={<Checkbox />} />))
+          .assertion(Interactor({ checked: true }).exists())
+      )
+      .child("by indeterminate", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="indeterminate" control={<Checkbox indeterminate />} />))
+          .assertion(Interactor({ indeterminate: true }).exists())
+      )
+      .child("by disabled", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="disabled" disabled control={<Checkbox />} />))
+          .assertion(Interactor({ disabled: true }).exists())
+      )
+      .child("by visible", (test) =>
+        test.step("render", () => render(<Checkbox />)).assertion(Interactor({ visible: false }).exists())
+      )
+  )
+  .child("click", (test) =>
+    test
+      .step("render", () => render(<FormControlLabel label="checkbox" control={<Checkbox />} />))
+      .child("default", (test) => test.assertion(Interactor("checkbox").is({ checked: false, focused: false })))
+      .child("click", (test) =>
+        test
+          .step("click", () => Interactor("checkbox").click())
+          // TODO `focused` doesn't set :(
+          .assertion(Interactor("checkbox").is({ checked: true /*, focused: true */ }))
```

Comment:
I think in places like right here where you are called `Interactor("checkbox")` you can do something like `Interactor()` `Interactor().is({ checked: true /*, focused: true */}))`. Since you are extending the checkbox interactor I believe the selector and locator will default to looking for a checkbox.